tests/tests_suite_test.go

```diff
@@ -76,6 +76,8 @@ func startVM(iso string) (context.Context, VM) {
 	stateDir, err := os.MkdirTemp("", "stateDir-*")
 	Expect(err).ToNot(HaveOccurred())
 
+	fmt.Printf("Statedir: %s\n", stateDir)
```

Comment:
leftover?

Reply:
no, this is useful to know where to find the serial.log when running the tests :)